portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path

```diff
@@ -90,6 +90,16 @@
 	<td>//div[contains(text(),'${columnName}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>COLUMN_RESIZER</td>
+	<td>//tr//th[contains(.,'${columnName}')]//*[@class="dnd-th-resizer is-allowed"]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>TABLE_COLUMN_NAME</td>
+	<td>//table//th[contains(text(),'${columnName}')] | //table//th//a[contains(.,'${columnName}')]</td>
+	<td></td>
+</tr>
```

Comment:
Does this work for FDS with and without the ClayTable applied?

Also, I initially assumed we'd want to put these paths in ClayTable.path. But I can also see the argument to keep the path here since FDS may be more discoverable for some people. I just want to make sure this is compatible for before and after ClayTable is applied.

Reply:
I let me test it without the current FF.

Reply:
It's weird because the path I didn't create does not work without the FF   `FrontendDataSet#TABLE_COLUMN_NAME`

Reply:
But I think it's better to move the paths I created to `ClayTable.path`, it looks more logical since we intend to apply it to all tables, right?